app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java

```diff
@@ -81,6 +81,7 @@ public void onStart() {
         setupRecyclerView();
         setupRefreshListener();
         getPresenter().attach(eventId, this);
+        getPresenter().loadTickets(true);
```

Comment:
It should be working automatically. Test why the Observable is not working correctly

Reply:
Sure